packages/hoppscotch-common/src/services/spotlight/searchers/environment.searcher.ts

```diff
@@ -318,6 +363,31 @@ export class SwitchEnvSpotlightSearcherService
           }
         })
       )
+
+      const workspace = this.workspaceService.currentWorkspace
+
+      if (workspace.value?.type === "team") {
+        this.fetchTeamEnvironmentList(workspace.value.teamID, (results) => {
```

Comment:
Would it be possible to use `async/await` here? I can see that `createSearchSession` is expected to be synchronous from the signature.